### DISCUSSION TITLE
Projects beta

### DISCUSSION THREAD

**Discussion by rastaman111** [community/community#14113] [Projects and Issues]
Good afternoon!  

I'm learning and working with the new Projects beta Api and I've run into a problem: how can I change the ordinal index of an item in a specific column (eg "ToDo")?  

Through updateProjectNextItemField, I did not figure out how to do it :(

**Comment by chanakyabhardwajj** [comment]
Hi

Could you please explain what you mean by the ordinal index of an item? Are you trying to change the position of an item in the project? The `updateProjectNextItemField` mutation currently does not support updating the position of an item.

**Reply by rastaman111** [reply]
Hi
Yes, updating the position of an item.

Is this feature coming or not planned?

**Reply by chanakyabhardwajj** [reply]
Yes we will support it, but do not have a fixed date yet.

**Reply by rastaman111** [reply]
Great, thanks for the feedback

**Reply by rastaman111** [reply]
please see this topic [Add item](https://github.com/github/feedback/discussions/14235)

We could not find how to add a new item to the desired field

**Comment by 1SwiftKey1** [comment]
Hello. First of all, you need to calm down and inhale and exhale. And the answer will be in front of you as soon as you understand the balance, removing the weight of these gray days. Just believe and everything will be fine!
